flink-yarn/src/main/java/org/apache/flink/yarn/YarnClusterDescriptor.java

```diff
@@ -941,10 +950,13 @@ private ApplicationReport startAppMaster(
 		// https://github.com/apache/hadoop/blob/trunk/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-site/src/site/markdown/YarnApplicationSecurity.md#identity-on-an-insecure-cluster-hadoop_user_name
 		appMasterEnv.put(YarnConfigKeys.ENV_HADOOP_USER_NAME, UserGroupInformation.getCurrentUser().getUserName());
 
-		if (remotePathKeytab != null) {
-			appMasterEnv.put(YarnConfigKeys.KEYTAB_PATH, remotePathKeytab.toString());
+		if (localizedKeytabPath != null) {
```

Comment:
I think we wouldn't have to do this if we simply used the Flink configuration for getting the config values. I think we don't need the environment variables at all and can thus simplify this.

Reply:
+1, reverting

Reply:
Just a minor question here. It seems user have to config `KERBEROS_LOGIN_KEYTAB`. Does it make sense? If the keytab is pre-installed, I think there is no need to define `KERBEROS_LOGIN_KEYTAB` anymore.

Reply:
we are not operating under the premise that only one Keytab file is pre-installed. multiple keytab files could be pre-installed and thus user would need to choose

Reply:
It seems user could choose it by `LOCALIZED_KEYTAB_PATH`. Am I understand it correctly? If `SHIP_LOCAL_KEYTAB` is false, then we do not care about the value of `KERBEROS_LOGIN_KEYTAB` but we still need to set it?